extensions/cornerstone/package.json

```diff
@@ -45,7 +45,8 @@
     "prop-types": "^15.6.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-resize-detector": "^6.7.6"
+    "react-resize-detector": "^6.7.6",
+    "ulog": "^1.1.0"
```

Comment:
We should consider reverting this (and related changes) and revisiting logging later.